addons/website_slides/security/website_slides_security.xml

```diff
@@ -140,7 +140,9 @@
                             ('channel_id.website_published', '=', True),
                     '|',
                         '&amp;',
+                            '|',
                                 ('channel_id.visibility', 'in', ('public','connected')),
+                                ('channel_id.partner_all_ids', 'in', user.partner_id.ids),
```

Comment:
There is already a condition on partner_ids a bit below ... are those two necessary ? ... to think again with more :coffee: maybe.

Reply:
yes because we give access 'as if it were public' to logged invited partners. -> partner_ALL_ids. (previews / categories).

But we give access to all slides to enrolled partners. -> partner_ids. :)

Reply:
Mmmh ... complicated stuff for the ORM :grimacing: